Maths/AverageMedian.js

```diff
@@ -9,22 +9,14 @@
  */
 
 const averageMedian = (sourceArrayOfNumbers) => {
-  let numbers = [...sourceArrayOfNumbers]
-  let median = 0
+  const numbers = [...sourceArrayOfNumbers].sort(sortNumbers)
   const numLength = numbers.length
-  numbers = numbers.sort(sortNumbers)
 
   if (numLength % 2 === 0) {
-    median = (numbers[numLength / 2 - 1] + numbers[numLength / 2]) / 2
-  } else {
-    median = numbers[(numLength - 1) / 2]
-  }
-
-  return median
+    return (numbers[numLength / 2 - 1] + numbers[numLength / 2]) / 2
+  } else return numbers[Math.floor(numLength / 2)]
 }
 
-const sortNumbers = (num1, num2) => {
-  return num1 - num2
-}
+const sortNumbers = (num1, num2) => num1 - num2
```

Comment:
This way of sorting numbers is sketchy to be honest (JS doesn't have overflows, but float precision issues may result in this treating inequal numbers as equal even if float comparison would yield the correct result), but it is out of scope of this PR to fix it.

Reply:
i can make another pr after this and can handle that

Reply:
we can do like this to solve that 
```js 
const averageMedian = (sourceArrayOfNumbers) => {
  const numbers = [...sourceArrayOfNumbers].sort((a, b) => sortNumbers(a, b));
  const numLength = numbers.length;

  return numLength % 2 === 0
    ? (numbers[numLength / 2 - 1] + numbers[numLength / 2]) / 2
    : numbers[Math.floor(numLength / 2)];
};

const sortNumbers = (num1, num2) => num1 - num2;
```